### PR TITLE
KBV703_search_for_user_in_Experian_DB_via_user's_name 

### PR DESCRIPTION
**Pull request by chinmayi-gds**
This branch consists of the tests for https://govukverify.atlassian.net/browse/KBV-703  
'Search for user in Experian DB via user's name '  

The following test scenarios have been added :
1)User Search By UserName User Journey Happy Path (STUB)
2)User Search By Name That Does Not Exist In Experian Table(STUB)
3)User Search when no user entered on the search field(STUB)
4)User Search By Invalid UserName(STUB)

Changes have been made to the following :  
src/test/resources/features/FraudCRI.feature  
src/test/java/gov/di_ipv_fraud/step_definitions/FraudStepDefs.java src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java

### CODE REVIEW

File: src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java [hunk 1]
```diff
@@ -137,4 +156,59 @@ public void whoWeCheckDetailsWith(String page) {
...
+        usersearchButton.click();
+    }
+
+    public void goTofraudCRILink() {
+        fraudCRILink.click();
+    }
+
+    public void userNameInJsonResponse() throws JsonProcessingException {
+        String result = JSONPayload.getText();
+        System.out.println("result = " + result);
```
Comment (gds-jbentham) [inline src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java:177]: Can we switch these system.out.println lines to use the logger like in the configuration service
    private static final Logger LOGGER = LogManager.getLogger();
It makes these much easier to read in an environment if we need to

Comment (chinmayi-gds) [reply src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java]: @gds-jbentham As requested, I have removed the System.out.println lines and have now used logger instead.

File: src/test/resources/features/FraudCRI.feature [hunk 2]
```diff
@@ -36,3 +36,63 @@ Feature: Fraud CRI
...
+    And The test is complete and I close the driver
+
+    Examples:
+      | environment |
+      | Build       |
+      | Staging     |
+      | Integration |
+
+  @userSearch_by_non_existent_name
+  Scenario Outline: User Search By Name That Does Not Exist In Experian Table(STUB)
```
[the remark this answers is not in the document]
Comment (chinmayi-gds) [reply src/test/resources/features/FraudCRI.feature:57]: @gds-jbentham In the Scenario Outline, I have included STUB in brackets to specify that these are tests of STUB.
Basically, I wanted to capture the 'No matches' error messages in these tests. Do you reckon I remove these tests?

[the remark this answers is not in the document]
Comment (chinmayi-gds) [reply src/test/resources/features/FraudCRI.feature]: @gds-jbentham I have now removed the tests of the stub @userSearch_by_non_existent_name and @userSearch_by_no_user. Are you able to take another look at the pull request please?

**Review by gds-jbentham — APPROVED** [review]

**Pull request merged**